compaction/compaction_manager.hh

```diff
@@ -299,6 +299,7 @@ private:
     utils::updateable_value<float> _static_shares;
     serialized_action _update_compaction_static_shares_action;
     utils::observer<float> _compaction_static_shares_observer;
+    uint64_t validation_errors = 0;
```

Comment:
Since this is a private member out convention is to start it with `_`, i.e. `_validation_errors`.